cmd/opfgd/start.go

```diff
@@ -85,22 +88,15 @@ func runStartCmd(ctx client.Context, cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	// Set up channel to listen for interrupt signal
-	sigChan := make(chan os.Signal, 1)
-	signal.Notify(sigChan, os.Interrupt, syscall.SIGTERM)
```

Comment:
why this is not needed now?

Reply:
It is replaced with a more common way and reuses the `shutdownInterceptor`
```
<-shutdownInterceptor.ShutdownChannel()
```